Doc/library/graphlib.rst

```diff
@@ -126,35 +128,36 @@
           if ts:
               ...
 
-      Raises :exc:`ValueError` if called without calling
-      :meth:`~TopologicalSorter.prepare` previously.
+      Raises :exc:`ValueError` if this method is called before
+      :meth:`~TopologicalSorter.prepare`.
 
    .. method:: done(*nodes)
 
-      Marks a set of nodes returned by :meth:`TopologicalSorter.get_ready` as
-      processed, unblocking any successor of each node in *nodes* for being
-      returned in the future by a call to :meth:`TopologicalSorter.get_ready`.
+      Marks a set of nodes returned by :meth:`~TopologicalSorter.get_ready` as
+      processed, unblocking any successors of each node in *nodes* for being
```

Comment:
I am used to seeing _any_ used in the singular, like _each_ or _every_.  _all_ is plural.

Reply:
+1, in this case it should say *all*, or maybe just a generic *the*.

Then it would read: `... unblocking the successors of each node in *nodes* for ...`